Update USBDevices.md for MyTeletouch mouse, keyboard and gamepad from your phone

julianls opened the pull request:
Hi, I'm not sure that this is the best way to tell that I have successfully tested MyTeletouch as wireless mouse and keyboard from phone for RPI2 running Windows 10 IoT core. Also works great with Retropie running on RPI2 as a gamepad.

julianls committed:
Update USBDevices.md

Hi, I'm not sure that this is the best way to tell that I have successfully tested MyTeletouch as wireless mouse and keyboard from phone for RPI2 running Windows 10 IoT core. Also works great with Retropie running on RPI2 as a gamepad.

msftclas commented:
Hi **@julianls**, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no Contribution License Agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, MSBOT;

HerrickSpencer commented on en-US/win10/SupportedInterfaces/USBDevices.md:
```diff
@@ -52,4 +52,5 @@ Chalkboard Electronics 7" LCD Capacitive Touchscreen Display | RPI2
 Vodafone (Huawei) K5150                                      | RPI2, MBM         | Vodafone (Huawei) K5150 150Mbps 4G LTE FDD USB Mobile Broadband Modem                                                                                                                 |                                                                                                                                      ...
 Sierra Wireless Beam (AirCard 340U)                          | MBM               | Sierra Wireless Beam (AirCard 340U) 4G LTE USB Mobile Broadband Modem                                                                                                                 |                                                                                                                                      ...
 Microsoft Xbox 360 Controller                                | RPI2              | An HID-compliant USB gamepad for Microsoft's Xbox 360                                                                                                                                 | Access using [HidDevice]('https://msdn.microsoft.com/en-us/library/windows/apps/windows.devices.humaninterfacedevice.hiddevice.aspx' ...
+MyTeletouch                                | RPI2              | An HID-compliant USB wireless mouse, keyboard and gamepad                                                                                                                                 | Access using your phone (Iphone, Android, Windows)                                                                           | [Product Webpage](h ...
```
Thanks for the update Julianls!  
Are you a Microsoft employee?  If not, please remove the "MSVerified" Icon till we can get a chance to test this internally.
The link you are using for myteletouch should not be secured with https... can you simply remove the 's'?
Thanks!

julianls replied:
Hi Thanks for your prompt response. I'm not MS employee so I have removed '![Verified][MSVerified]{:.MsVerified} ' and -s from the https. I will be happy to send one piece of MyTeletouch if someone at MS is interested to play with it and test.

Regards
Julian

julianls committed:
Update USBDevices.md for MyTeletouch

Update USBDevices.md for MyTeletouch mouse, keyboard and gamepad from your phone. 
Removed '![Verified][MSVerified]{:.MsVerified}' and changed https to http
